File: 0026-Remove-Duplicates-from-Sorted-Array/solution.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        idx = 0
        numsSize = len(nums)
        for i in range(numsSize):
            if nums[idx] != nums[i]:
                idx += 1
                nums[idx] = nums[i]

        return idx+1
        

File: 0026-Remove-Duplicates-from-Sorted-Array/solution.c
int removeDuplicates(int* nums, int numsSize) {
    int idx = 0;
    for (int i = 1; i < numsSize; i++) {
        if (nums[idx] != nums[i]) {
            idx++;
            nums[idx] = nums[i];
        }
    }
    
    return idx + 1;
}
